swift-sdk/Internal/EmbeddedMessagingManager.swift

```diff
@@ -157,7 +157,7 @@ class EmbeddedMessagingManager: NSObject, IterableEmbeddedMessagingManagerProtoc
             apiClient.track(embeddedMessageReceived: message)
 //            IterableAPI.track(event: "embedded-messaging",
 //                              dataFields: ["name": "received",
-//                                           "messageId": message.metadata.id]
+//                                           "messageId": message.metadata.messageId]
```

Comment:
Seems like this section needs to be cleaned up on a separate PR. Looks like a function was added in the impression tracking PR.